var/spack/repos/builtin/packages/sqlite/package.py

```diff
@@ -285,6 +285,23 @@ class NMakeBuilder(spack.build_systems.nmake.NMakeBuilder):
     def makefile_name(self):
         return "Makefile.msc"
 
+    def nmake_args(self):
+        return [
+            "USE_NATIVE_LIBPATHS=1",
+            "DYNAMIC_SHELL=1",
+            "OPTS="
+            "-DSQLITE_ENABLE_FTS3=1 "
+            "-DSQLITE_ENABLE_FTS4=1 "
+            "-DSQLITE_ENABLE_FTS5=1 "
```

Comment:
This makes it look like we could condition this on the `+fts` variant (which right now is not available on Windows).

Reply:
Good point!